build-user-docs/skeletal-docs/temper-docs/docs/index.md

```diff
@@ -28,7 +28,7 @@ programming languages.
 
 <div style="display:inline-block; max-width: 20em; vertical-align: middle; text-align: left" markdown="1" class="transparent-code-root">
 
-```temper
+```temper {.no-copy}
```

Comment:
The copy-to-clipboard icon is overly intrusive as this small code sample is in a fixed-size box and the icon is not invisible when not hovered, just de-emphasized